Atomic/trust.py

```diff
@@ -63,6 +63,8 @@ def cli(subparser):
     deletep.add_argument("--sigstoretype", dest="sigstoretype", default="web",
                          choices=['atomic', 'local', 'web'],
                          help=sigstore_help)
+    deletep.add_argument("--save-sigstore", dest="savesigstore", default=True, action="store_false",
```

Comment:
The only way I see to make this work is to use `default=True, action="store_false"`. Any other combination I've tested does not produce the behavior we document and expect.